h/static/scripts/directive/excerpt.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = ->
+  link: (scope, elem, attr, ctrl) ->
+    scope.$evalAsync -> if elem[0].scrollHeight > elem[0].clientHeight
+      elem.prepend angular.element '<span class="more"> More...</span>'
+      elem.append angular.element '<span class="less"> Less ^</span>'
+      elem.find('.more').on 'click', ->
+        $(this).hide()
+        elem.addClass('show-full-excerpt')
```

Comment:
Might be better to namespace all these in a typical style: `excerpt`, `excerpt-reveal` (or whatever word you like), `excerpt-show-more`, `excerpt-show-less`, or something like this.